src/http/conf.rs

```diff
@@ -31,3 +31,35 @@ pub unsafe fn ngx_http_conf_get_module_loc_conf(
     let http_conf_ctx = (*cf).ctx as *mut ngx_http_conf_ctx_t;
     *(*http_conf_ctx).loc_conf.add(module.ctx_index) as *mut ngx_http_core_loc_conf_t
 }
+
+/// # Safety
+///
+/// The caller has provided a value `ngx_http_upstream_srv_conf_t. If the `us` argument is null, a
+/// None Option is returned; however, if the `us` internal fields are invalid or the module index
+/// is out of bounds failures may still occur.
+pub unsafe fn ngx_http_conf_upstream_srv_conf_immutable<T>(
```

Comment:
seems to me based on this https://doc.rust-lang.org/std/primitive.pointer.html we can name as `ngx_http_conf_upstream_srv_conf_as_ptr` or 

some ref: https://doc.rust-lang.org/src/core/ptr/const_ptr.rs.html#1629